src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.UnitTestingLowPriorityProcessor.cs

```diff
@@ -152,12 +152,19 @@ private async Task ProcessProjectAsync(ImmutableArray<IUnitTestingIncrementalAna
                                 if (project != null)
                                 {
                                     var reasons = workItem.InvocationReasons;
+#if false // Not used in unit testing crawling
                                     var semanticsChanged = reasons.Contains(UnitTestingPredefinedInvocationReasons.SemanticChanged) ||
```

Comment:
SemanticsChanged is one of the used reasons. But perhaps there is some other reason why this is unused.

Reply:
Ah never mind - looks like this is for the AnalyzeProject API where the semanticsChanged flag is unused?

Reply:
correct :)